d3a_interface/__init__.py

```diff
@@ -0,0 +1,5 @@
+import sys
+
+import gsy_framework  # NOQA
+
+sys.modules[__name__] = sys.modules['gsy_framework']
```

Comment:
Could you explain what this code is for? Will users be able to still use `import d3a_interface`?

Nit:
```suggestion
import gsy_framework

sys.modules[__name__] = sys.modules["gsy_framework"]
```

Reply:
For backwards compatibility. The main rationale behind this is for the users that have setup scripts to still continue to use them with the old imports.